libscanbuild/intercept.py

```diff
@@ -154,6 +154,19 @@ def intercept_compiler_wrapper(_, execution):
         logging.warning(message_prefix, 'io problem')
 
 
+def expand_cmd_with_response_files(cmd):
+    for param in cmd[:]:
+        if param[:1] == '@' and os.path.isfile(param[1:]):
+            # can throw but intercap_compiler_wrapper should catch
+            with open(param[1:], 'r') as f:
```

Comment:
Could you extract this into a method too? :) What I see here is that reading the file and change the list element at the same place. Maybe to read the file and return the stripped content would deserve a separate method. What do you think? It could also placeholder to a comment which describes the file format of these files.